metadata-models/src/main/pegasus/com/linkedin/metadata/aspect/DatasetAspect.pdl

```diff
@@ -24,5 +25,6 @@ typeref DatasetAspect = union[
   Status,
   SchemaMetadata,
   EditableSchemaMetadata,
-  GlobalTags
+  GlobalTags,
+  GlossaryTerms
```

Comment:
are there other entities aside from dataset we want to add this aspect to? Charts, dashboards, datajobs or dataflows?

Reply:
As of now we are only linking the business term to datasets